acstools/acszpt.py

```diff
@@ -319,6 +310,15 @@ def _parse_and_format(self):
         # Remove the units attached to PHOTFLAM and PHOTPLAM column names.
         td = [val.text.split(' ')[0] for val in td]
 
+        # Grab warnings returned by the ZPT calc as marked by stylized h4 tag
+        warnings = soup.find_all('h4', {'style': "color:red;"})
+
+        # Remove tags and 'Warnings: ' label attached to warnings
+        str_warnings = [w.get_text().replace('Warnings: ', '') for w in warnings]
+
+        # Add non-empty strings to _warnings attribute
+        self._warnings = [w for w in str_warnings if w.strip()]
```

Comment:
I feel like this all can be collapsed into one line. But I don't understand what kind of warnings the server is going to issue to give a recommendation.

Reply:
It can be, but it just looked messy as one line. I have it this way just in case there are multiple warning lines passed, but I'm not well versed in the possible output of the ZPT tool

Reply:
Okay, let's leave it as-is then. 😸